app/models/provisioning_template.rb

```diff
@@ -121,7 +121,8 @@ def self.build_pxe_default(renderer)
     if error_msg.empty?
       begin
         @profiles = pxe_default_combos
-        menu = renderer.render_safe(default_template.template, [:default_template_url], {:profiles => @profiles})
+        allowed_helpers = ALLOWED_GENERIC_HELPERS + [ :default_template_url ]
```

Comment:
I receive an error building the default PXE menu, I think from here: `failed to process template: uninitialized constant ProvisioningTemplate::ALLOWED_GENERIC_HELPERS`